src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -186,7 +186,19 @@ export async function containerCreate(operation: Operation, script: string, comm
             args = [projectLocation, LOCAL_WORKSPACE, operation.projectInfo.projectID, command, operation.containerName,
                 String(operation.projectInfo.autoBuildEnabled), logName, operation.projectInfo.startMode, operation.projectInfo.debugPort,
                 (operation.projectInfo.forceAction) ? String(operation.projectInfo.forceAction) : "NONE", logDir, deploymentRegistry, userMavenSettings];
-    }
+        } else if (projectType == "odo") {
```

Comment:
Why do we need to do this?  Does Appsody has equivalent?  We should try to avoid burning extension specific code into PFE.  Can this be move to part of the extension code?